clients/python/basic.sender.py

```diff
@@ -10,6 +11,8 @@ def __init__(self):
         super(BasicSender, self).__init__()
         # Sender specifics
         self._sender = None
+        self._released_tags = []
```

Comment:
This list grows unlimitedly right? What happen if it grows too much? probably something crashes...

Reply:
I have already calculated that. The value should be minimal.
In tests like using 100k messages, we usually see one or two messages only
being added to this list.
In a worst possible scenario, if all 100k msgs are released, then this list will use 3.5 MB of memory (per sender instance).

I would say not a big deal.